### ISSUE TITLE
Error Document is not defined - Cloudinary React on Server Side Rendering 

### ISSUE THREAD

**Issue by saddamhbb** [cloudinary/cloudinary-react#101]
Hi there cloudinary team :)

I got this error when implement cloudinary on server side rendering:
(node:40320) UnhandledPromiseRejectionWarning: ReferenceError: document is not defined.

Is there any workaround to fix this? 

Thanks.

**Comment by saddamhbb** [comment]
Seems like I already fixed it.

For anyone who stuck with the same problem like me, if you want to call the <Image /> or <Video /> component you need to make sure your component already mounted.

So you need to create a initial state didMount = false, and when component already mounted, call the this.setState to set didMount = true in componentDidMount lifecycle.

And in your render life you can do something like this:

```
render() {
   return (
     {
        this.state.didMount && <Image />
     }
   )
}
```

**Comment by strausr** [comment]
@saddamhbb Thanks for the input!

**Comment by saddamhbb** [comment]
@strausr Sure, Anytime :)

**Comment by roeeba** [comment]
Closing this issue as it's not related to our React package. We will update here regardless once the feature is ready for use.

**Comment by roeeba** [comment]
Closing this issue due to the time elapsed. Please feel free to either re-open the issue, contact our support at http://support.cloudinary.com or create a new ticket if you have any additional issues.
